components/count-badge/count-badge-icon.js

```diff
@@ -33,23 +33,35 @@ class CountBadgeIcon extends CountBadgeMixin(LitElement) {
 			display: inline-block;
 		}
 
+		.d2l-count-badge-number-wrapper {
+			position: absolute;
+			width: max-content;
```

Comment:
This line allows us to overflow. Removing it, and we get the stacked option for multi-word strings.